crates/api/src/kitsune.rs

```diff
@@ -5,6 +5,18 @@ use std::sync::Arc;
 
 /// Handler for events coming out of Kitsune2.
 pub trait KitsuneHandler: 'static + Send + Sync + std::fmt::Debug {
+    /// A notification that a new listening address has been bound.
+    /// Peers should now go to this new address to reach this node.
+    fn new_listening_address(&self, this_url: Url) {
+        drop(this_url);
```

Comment:
What are these `drop`s for?

Reply:
Underscores at the beginning of parameters show up in the docs, so I don't like them there. The explicit drops prevent the unused parameter warning. These default implementations don't need to do anything.

Reply:
But you want a default implementation that does nothing, I guess. Alright, got it.